projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Filesystem_Utils.php

```diff
@@ -112,8 +112,9 @@ public static function get_request_filename( $parameters ) {
 		 * @param array $parameters The array of components, url, cookies, get parameters, etc.
 		 *
 		 * @since   1.0.0
+		 * @deprecated $$next-version$$
 		 */
-		$key_components = apply_filters( 'boost_cache_key_components', $parameters );
+		$key_components = apply_filters_deprecated( 'boost_cache_key_components', $parameters, '$$next-version$$', 'jetpack_boost_cache_parameters' );
```

Comment:
According to [`apply_filter_deprecated` docs](https://developer.wordpress.org/reference/functions/apply_filters_deprecated/#:~:text=Note%3A%20the%20value,array.%20For%20example%3A), the arguments need to be wrapped in an array.

```suggestion
		$key_components = apply_filters_deprecated( 'boost_cache_key_components', array( $parameters ), '$$next-version$$', 'jetpack_boost_cache_parameters' );
```